lib/sinon/util/fake_server.js

```diff
@@ -74,8 +74,10 @@ var fakeServer = {
             this.xhr = sinon.useFakeXMLHttpRequest();
         }
         server.requests = [];
-
         this.xhr.onCreate = function (xhrObj) {
+            xhrObj.unsafeHeadersEnabled = function () {
```

Comment:
Why does `unsafeHeadersEnabled` need to be a function - why can't it just be a bool?